pylabrobot/liquid_handling/liquid_handler.py

```diff
@@ -761,8 +761,8 @@ async def aspirate(
 
     extras = self._check_args(self.backend.aspirate, backend_kwargs,
       default={"ops", "use_channels"})
-    for extra in extras:
-      del backend_kwargs[extra]
+    # for extra in extras:
+    #   del backend_kwargs[extra]
```

Comment:
@rickwierenga how do you feel about deleting these in asp and dsp? otherwise, backend_kwargs are never passed to the chatterbox

Reply:
fixed with https://github.com/PyLabRobot/pylabrobot/pull/208/commits/60ba3e957436a4d5dab7f910a9f52df8027a92d2